src/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Index Page</title>
+    <link rel="stylesheet" href="dist/main.css">
+
+</head>
+<body>
+    <div class="container">
+        <h1>LeaderBoard</h1>
+        <section class="grid">
+            <div class="grid-item">
+                <div class="d-flex justify-content-between align-items-center">
+                    <h2 class="mb-0">Recent scores</h2>
+                    <div class="ms-3">
+                        <button id="btn" class="btn btn-primary">Refresher</button>
+                    </div>
+                </div>
+            
+                <div class="list-container">
+                    <ul id="list">
+                        <li>name 1</li>
+                        <li>name 2</li>
+                        <li>name 3</li>
+                        <li>name 4</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="grid-item right">
+                <h2>Add your score </h2>
+                <form id="form">
+                    <div class="input-container">
+                  
+                        <input type="text" id="input1" name="input1" placeholder="Enter text" required>
+                    </div>
+                    <div class="input-container">
+             
+                        <input type="password" id="input2" name="input2" placeholder="Enter password" required>
+                    </div>
+                    <input type="submit" value="Submit" class="submit-button">
```

Comment:
- [ ] When click on the "submit" button nothing happen. Kindly make sure the form "Submit" button save a score for a game created by you (sending data to the API) as requested in the project requirement.
![Screenshot from 2023-05-25 12-28-04](https://github.com/kayjeee/leaderboard-v2/assets/91142432/2a527465-0f61-489a-8cc3-bdf3342a6236)